tests/test_init.py

```diff
@@ -42,3 +42,17 @@ def test_no_seed():
         if np.all(n_priored > 0):
             return
     raise ValueError(f"Error getting all priors in {n_test_max} iterations.")
+
+
+def test_model_seed():
+    n_test = 4
+    seed = 192874123
+    last_train_idx = None
+    for _ in range(n_test):
+        reviewer = get_reviewer(
```

Comment:
Is there anything non-deterministic in this model and query strategy?

Reply:
Yes, the balance strategy (probably). Anyway, good point, I changed it to make it random for sure.